static/global.css

```diff
@@ -18,6 +18,8 @@
 
   --perma-light: #f8f8f8;
 
+  /* Generator: https://xgkft.csb.app/ (280px-1000px viewport-width, 1rem=16px) */
```

Comment:
I feel like I already reviewed these changes in #468. I am confused as to why that PR didn't get merged.

Reply:
it's merged into here because i figured "since im here at the blog i may as well work on the logo" only to realize karni worked on it already so I just merged hers into mine and changed some things

Reply:
> it's merged into here because i figured "since im here at the blog i may as well work on the logo" only to realize karni worked on it already so I just merged hers into mine and changed some things

If your intention is to build off of #468, we would normally base your `blog-tweaks` branch on the `fix/430` branch. Then once #468 is merged, your `blog-tweaks` branch should contain only the changes you’ve added when you compare it with the `main` branch.

Reply:
Hm, I'll just reopen it then